src/components/EditableTypography/HiddenInputPlaceholder.tsx

```diff
@@ -1,26 +1,33 @@
-import React, { FC, useLayoutEffect, useRef } from "react";
+import React, { useLayoutEffect, useRef, forwardRef } from "react";
 import cx from "classnames";
 import styles from "./HiddenInputPlaceholder.module.scss";
+import useMergeRefs from "../../hooks/useMergeRefs";
 
 interface HiddenInputPlaceholderProps {
   className?: string;
   value: string;
   onChange: (width: number) => void;
 }
 
-const HiddenInputPlaceholder: FC<HiddenInputPlaceholderProps> = ({ className, value, onChange }) => {
-  const valueRef = useRef(null);
+const HiddenInputPlaceholder = forwardRef<HTMLElement, HiddenInputPlaceholderProps>(
+  ({ className, value, onChange }, ref) => {
+    const valueRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, valueRef] });
 
-  useLayoutEffect(() => {
-    const { width } = valueRef.current.getBoundingClientRect();
-    onChange(width);
-  }, [onChange, value]);
+    useLayoutEffect(() => {
+      if (!valueRef.current) {
```

Comment:
what's the motivation of using `valueRef`/`mergedRef` here instead?

Reply:
What I mean is, can't you use only the `mergedRef` here?

Reply:
I might be mistaken but there's a bug in the return value of useMergedRefs that causes some typing issues, I checked quickly and it looks like it